pkgs/tools/text/ansifilter/default.nix

```diff
@@ -0,0 +1,26 @@
+{ fetchurl, stdenv, pkgconfig, boost, lua }:
+let version = "1.15";
+    pkgsha = "65dc20cc1a03d4feba990f830186404c90462d599e5f4b37610d4d822d67aec4";
+in stdenv.mkDerivation {
+  name = "ansifilter-${version}";
+  buildInputs = [
+    pkgconfig boost lua
+  ];
+  src = fetchurl {
+    url = "http://www.andre-simon.de/zip/ansifilter-${version}.tar.bz2";
+    sha256 = pkgsha;
+  };
+  meta = {
+    description = ''
```

Comment:
Minor nitpick: please remove the newline at the start here; if you want to provide a longer form description, use `longDescription`.